<!DOCTYPE html>
<!-- START: inst/pkgdown/templates/layout.html --><!-- Generated by pkgdown: do not edit by hand --><html lang="en" data-bs-theme="auto">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<title>Lesson Title: All in One View</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<script src="../assets/themetoggle.js"></script><link rel="stylesheet" type="text/css" href="../assets/styles.css">
<script src="../assets/scripts.js" type="text/javascript"></script><!-- mathjax --><script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      config: ["MMLorHTML.js"],
      jax: ["input/TeX","input/MathML","output/HTML-CSS","output/NativeMML", "output/PreviewHTML"],
      extensions: ["tex2jax.js","mml2jax.js","MathMenu.js","MathZoom.js", "fast-preview.js", "AssistiveMML.js", "a11y/accessibility-menu.js"],
      TeX: {
        extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"]
      },
      tex2jax: {
        inlineMath: [['\\(', '\\)']],
        displayMath: [ ['$$','$$'], ['\\[', '\\]'] ],
        processEscapes: true
      }
    });
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><!-- Responsive Favicon for The Carpentries --><link rel="apple-touch-icon" sizes="180x180" href="../favicons/uom/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicons/uom/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="../favicons/uom/favicon-16x16.png">
<link rel="manifest" href="../favicons/uom/site.webmanifest">
<link rel="mask-icon" href="../favicons/uom/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="black">
</head>
<body>
    <header id="top" class="navbar navbar-expand-md top-nav uom"><svg xmlns="http://www.w3.org/2000/svg" class="d-none"><symbol id="check2" viewbox="0 0 16 16"><path d="M13.854 3.646a.5.5 0 0 1 0 .708l-7 7a.5.5 0 0 1-.708 0l-3.5-3.5a.5.5 0 1 1 .708-.708L6.5 10.293l6.646-6.647a.5.5 0 0 1 .708 0z"></path></symbol><symbol id="circle-half" viewbox="0 0 16 16"><path d="M8 15A7 7 0 1 0 8 1v14zm0 1A8 8 0 1 1 8 0a8 8 0 0 1 0 16z"></path></symbol><symbol id="moon-stars-fill" viewbox="0 0 16 16"><path d="M6 .278a.768.768 0 0 1 .08.858 7.208 7.208 0 0 0-.878 3.46c0 4.021 3.278 7.277 7.318 7.277.527 0 1.04-.055 1.533-.16a.787.787 0 0 1 .81.316.733.733 0 0 1-.031.893A8.349 8.349 0 0 1 8.344 16C3.734 16 0 12.286 0 7.71 0 4.266 2.114 1.312 5.124.06A.752.752 0 0 1 6 .278z"></path><path d="M10.794 3.148a.217.217 0 0 1 .412 0l.387 1.162c.173.518.579.924 1.097 1.097l1.162.387a.217.217 0 0 1 0 .412l-1.162.387a1.734 1.734 0 0 0-1.097 1.097l-.387 1.162a.217.217 0 0 1-.412 0l-.387-1.162A1.734 1.734 0 0 0 9.31 6.593l-1.162-.387a.217.217 0 0 1 0-.412l1.162-.387a1.734 1.734 0 0 0 1.097-1.097l.387-1.162zM13.863.099a.145.145 0 0 1 .274 0l.258.774c.115.346.386.617.732.732l.774.258a.145.145 0 0 1 0 .274l-.774.258a1.156 1.156 0 0 0-.732.732l-.258.774a.145.145 0 0 1-.274 0l-.258-.774a1.156 1.156 0 0 0-.732-.732l-.774-.258a.145.145 0 0 1 0-.274l.774-.258c.346-.115.617-.386.732-.732L13.863.1z"></path></symbol><symbol id="sun-fill" viewbox="0 0 16 16"><path d="M8 12a4 4 0 1 0 0-8 4 4 0 0 0 0 8zM8 0a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 0zm0 13a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 13zm8-5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2a.5.5 0 0 1 .5.5zM3 8a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2A.5.5 0 0 1 3 8zm10.657-5.657a.5.5 0 0 1 0 .707l-1.414 1.415a.5.5 0 1 1-.707-.708l1.414-1.414a.5.5 0 0 1 .707 0zm-9.193 9.193a.5.5 0 0 1 0 .707L3.05 13.657a.5.5 0 0 1-.707-.707l1.414-1.414a.5.5 0 0 1 .707 0zm9.193 2.121a.5.5 0 0 1-.707 0l-1.414-1.414a.5.5 0 0 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .707zM4.464 4.465a.5.5 0 0 1-.707 0L2.343 3.05a.5.5 0 1 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .708z"></path></symbol></svg><a class="visually-hidden-focusable skip-link" href="#main-content">Skip to main content</a>
  <div class="container-fluid top-nav-container">
    <div class="col-md-8">
      <div class="large-logo">
        <img id="uom-logo" alt="Lesson Description" src="../assets/images/uom-logo.svg"><span class="badge text-bg-danger">
          <abbr title="This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught.">
            <a href="https://docs.carpentries.org/resources/curriculum/lesson-life-cycle.html" class="external-link alert-link">
              <i aria-hidden="true" class="icon" data-feather="alert-octagon" style="border-radius: 5px"></i>
              Pre-Alpha
            </a>
            <span class="visually-hidden">This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught.</span>
          </abbr>
        </span>

      </div>
    </div>
    <div class="selector-container">
      <div id="theme-selector">
        <li class="nav-item dropdown" id="theme-button-list">
          <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
            <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="bd-theme-text">
<li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                Light
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                Dark
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                Auto
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
          </ul>
</li>
      </div>

      <div class="dropdown" id="instructor-dropdown">
        <button class="btn btn-secondary dropdown-toggle bordered-button" type="button" id="dropdownMenu1" data-bs-toggle="dropdown" aria-expanded="false">
          <i aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View <i data-feather="chevron-down"></i>
        </button>
        <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
<li><button class="dropdown-item" type="button" onclick="window.location.href='../aio.html';">Learner View</button></li>
        </ul>
</div>
    </div>
  </div>
  <hr></header><nav class="navbar navbar-expand-xl bottom-nav uom" aria-label="Main Navigation"><div class="container-fluid nav-container">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle Navigation">
      <span class="navbar-toggler-icon"></span>
      <span class="menu-title">Menu</span>
    </button>
    <div class="nav-logo">
      <img class="small-logo" alt="Lesson Description" src="../assets/images/uom-logo-sm.svg">
</div>
    <div class="lesson-title-md">
      Lesson Title
    </div>
    <div class="search-icon-sm">
      <!-- TODO: do not show until we have search
        <i role="img" aria-label="Search the All In One page" data-feather="search"></i>
      -->
    </div>
    <div class="desktop-nav">
      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
<li class="nav-item">
          <span class="lesson-title">
            Lesson Title
          </span>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/key-points.html">Key Points</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/instructor-notes.html">Instructor Notes</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/images.html">Extract All Images</a>
        </li>
        <li class="nav-item dropdown">
          <button class="nav-link dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-expanded="false">
            More <i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
<hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
          </ul>
</li>
      </ul>
</div>
    <!--
    <form class="d-flex col-md-2 search-form">
      <fieldset disabled>
      <input class="form-control me-2 searchbox" type="search" placeholder="" aria-label="">
        <button class="btn btn-outline-success tablet-search-button"  type="submit">
          <i class="search-icon" data-feather="search" role="img" aria-label="Search the All In One page"></i>
        </button>
      </fieldset>
    </form>
    -->
    <a id="search-button" class="btn btn-primary" href="../instructor/aio.html" role="button" aria-label="Search the All In One page">Search the All In One page</a>
  </div>
<!--/div.container-fluid -->
</nav><div class="col-md-12 mobile-title">
  Lesson Title
</div>

<aside class="col-md-12 lesson-progress"><div style="width: %" class="percentage">
    %
  </div>
  <div class="progress uom">
    <div class="progress-bar uom" role="progressbar" style="width: %" aria-valuenow="" aria-label="Lesson Progress" aria-valuemin="0" aria-valuemax="100">
    </div>
  </div>
</aside><div class="container">
      <div class="row">
        <!-- START: inst/pkgdown/templates/navbar.html -->
<div id="sidebar-col" class="col-lg-4">
  <div id="sidebar" class="sidebar">
      <nav aria-labelledby="flush-headingEleven"><button role="button" aria-label="close menu" alt="close menu" aria-expanded="true" aria-controls="sidebar" class="collapse-toggle" data-collapse="Collapse " data-episodes="Episodes ">
          <i class="search-icon" data-feather="x" role="img"></i>
        </button>
        <div class="sidebar-inner">
          <div class="row mobile-row" id="theme-row-mobile">
            <div class="col" id="theme-selector">
              <li class="nav-item dropdown" id="theme-button-list">
                <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
                  <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><span class="d-lg-none ms-1" id="bd-theme-text">Toggle Theme</span>
                </button>
                <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="bd-theme-text">
<li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                      Light
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                      Dark
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                      <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                      Auto
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                </ul>
</li>
            </div>
          </div>
          <div class="row mobile-row">
            <div class="col">
              <div class="sidenav-view-selector">
                <div class="accordion accordion-flush" id="accordionFlush9">
                  <div class="accordion-item">
                    <h2 class="accordion-header" id="flush-headingNine">
                      <button class="accordion-button collapsed" id="instructor" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseNine" aria-expanded="false" aria-controls="flush-collapseNine">
                        <i id="eye" aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View
                      </button>
                    </h2>
                    <div id="flush-collapseNine" class="accordion-collapse collapse" aria-labelledby="flush-headingNine" data-bs-parent="#accordionFlush2">
                      <div class="accordion-body">
                        <a href="../aio.html">Learner View</a>
                      </div>
                    </div>
                  </div>
<!--/div.accordion-item-->
                </div>
<!--/div.accordion-flush-->
              </div>
<!--div.sidenav-view-selector -->
            </div>
<!--/div.col -->

            <hr>
</div>
<!--/div.mobile-row -->

          <div class="accordion accordion-flush" id="accordionFlush11">
            <div class="accordion-item">

              <button id="chapters" class="accordion-button show" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseEleven" aria-expanded="false" aria-controls="flush-collapseEleven">
                <h2 class="accordion-header chapters" id="flush-headingEleven">
                  EPISODES
                </h2>
              </button>
              <div id="flush-collapseEleven" class="accordion-collapse show collapse" aria-labelledby="flush-headingEleven" data-bs-parent="#accordionFlush11">

                <div class="accordion-body">
                  <div class="accordion accordion-flush" id="accordionFlush1">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading1">
        <a href="index.html">Summary and Schedule</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush2">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading2">
        <a href="What%20is%20RNA-seq.html">1. Objectives</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush3">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading3">
        <a href="Why%20is%20a%20good%20experimental%20design%20vital.html">2. Why is a good experimental design vital?</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush4">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading4">
        <a href="Terminology.html">3. Terminology</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush5">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading5">
        <a href="The%20importance%20of%20replicates%20to%20estimate%20variance.html">4. The importance of replicates to estimate variance</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush6">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading6">
        <a href="How%20many%20replicates%20and%20how%20many%20reads%20do%20I%20need.html">5. How many replicates and how many reads do I need?</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush7">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading7">
        <a href="Sequencing%20options%20to%20consider.html">6. Sequencing options to consider</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush8">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading8">
        <a href="Summary.html">7. Summary</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

                </div>
              </div>
            </div>

            <hr class="half-width">
<div class="accordion accordion-flush lesson-resources" id="accordionFlush12">
              <div class="accordion-item">
                <h2 class="accordion-header" id="flush-headingTwelve">
                  <button class="accordion-button collapsed" id="lesson-resources" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseTwelve" aria-expanded="false" aria-controls="flush-collapseTwelve">
                    RESOURCES
                  </button>
                </h2>
                <div id="flush-collapseTwelve" class="accordion-collapse collapse" aria-labelledby="flush-headingTwelve" data-bs-parent="#accordionFlush12">
                  <div class="accordion-body">
                    <ul>
<li>
                        <a href="../instructor/key-points.html">Key Points</a>
                      </li>
                      <li>
                        <a href="../instructor/instructor-notes.html">Instructor Notes</a>
                      </li>
                      <li>
                        <a href="../instructor/images.html">Extract All Images</a>
                      </li>
                      <hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
                    </ul>
</div>
                </div>
              </div>
            </div>
            <hr class="half-width lesson-resources">
<a href="../instructor/aio.html">See all in one page</a>


            <hr class="d-none d-sm-block d-md-none">
<div class="d-grid gap-1">

            </div>
          </div>
<!-- /div.accordion -->
        </div>
<!-- /div.sidebar-inner -->
      </nav>
</div>
<!-- /div.sidebar -->
  </div>
<!-- /div.sidebar-col -->
<!-- END:   inst/pkgdown/templates/navbar.html-->

        <!-- START: inst/pkgdown/templates/content-extra.html -->
  <div class="col-xl-8 col-lg-12 primary-content">
    <main id="main-content" class="main-content"><div class="container lesson-content">
        
        
<section id="aio-What is RNA-seq"><p>Content from <a href="What%20is%20RNA-seq.html">Objectives</a></p>
<hr>
<p>Last updated on 2025-08-12 |

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/episodes/What%20is%20RNA-seq.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 65 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<p><input type="checkbox" unchecked> Explain what RNA-seq is and
describe its typical workflow.<br><input type="checkbox" unchecked> Understand why good experimental
design is crucial for RNA-seq analysis.<br><input type="checkbox" unchecked> Identify and define key RNA-seq
terminology:<br>
• <input type="checkbox" unchecked> Variability<br>
• <input type="checkbox" unchecked> Feature<br>
• <input type="checkbox" unchecked> Biological vs technical
replicates<br>
• <input type="checkbox" unchecked> Covariates and confounding
variables<br>
• <input type="checkbox" unchecked> Statistical power<br><input type="checkbox" unchecked> Describe strategies to minimise
variability and control confounding variables.<br><input type="checkbox" unchecked> Estimate appropriate replicate numbers
and sequencing depth for an experiment.<br><input type="checkbox" unchecked> Compare sequencing design
choices:<br>
• <input type="checkbox" unchecked> Poly-A enrichment vs
ribo-depletion<br>
• <input type="checkbox" unchecked> Single-end vs paired-end reads<br>
• <input type="checkbox" unchecked> Stranded vs unstranded
libraries<br><input type="checkbox" unchecked> Understand the role of multiplexing
and spike-in controls in RNA-seq.</p>
<section><h2 class="section-heading" id="what-is-rna-sequencing">What is RNA sequencing<a class="anchor" aria-label="anchor" href="#what-is-rna-sequencing"></a>
</h2>
<hr class="half-width">
<p>RNA-seq is a method of measuring gene expression using shotgun
sequencing. The process involves reverse transcribing RNA into cDNA,
then sequencing fragments on a high-throughput platform such as Illumina
to obtain a large number of short reads. For each sample, the reads are
then aligned to a genome, and the number of reads aligned to each gene
or feature is recorded.</p>
<p>A typical RNA-seq experiment aims to find differentially expressed
genes between two conditions (e.g. up and down-regulated genes in
knock-out mice compared to wild-type mice). RNA-seq can also be used to
discover new transcripts, splice variants, and fusion genes.</p>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</section></section><section id="aio-Why is a good experimental design vital"><p>Content from <a href="Why%20is%20a%20good%20experimental%20design%20vital.html">Why is a good experimental design vital?</a></p>
<hr>
<p>Last updated on 2025-08-12 |

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/episodes/Why%20is%20a%20good%20experimental%20design%20vital.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 65 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<p>An RNA-seq experiment produces high dimensional data. This means we
get a huge number of observations for a small number of samples. For
example, the expression of ~20,000 genes could be measured for 6 samples
(3 knock-out and 3 wild-type). A frequently used approach to analyse
RNA-seq data is to fit each gene to a linear model where for each of the
20,000 genes, parameters need to be estimated using a small number of
observations. To complicate matters, each measurement of gene expression
is comprised of a mix of biological signal and unwanted noise. Thus, in
order to perform a robust statistical analysis, the methodology must be
carefully designed.</p>
<p>Before you begin any RNA-seq experiment, some questions you should
ask yourself are:</p>
<ul>
<li>Why do you expect to find differentially expressed genes in the
particular tissue?</li>
<li>What types of genes do you expect to find differentially
expressed?</li>
<li>What are the sources of variability from your samples?</li>
<li>Where do you expect most of your variation to come from?</li>
<li>A coherent experimental design is the groundwork of a successful
experiment. You should invest time and thought in designing a robust
experiment as failing to think this step through can lead to unusable
data and wasted time, money, and effort.</li>
</ul>
<p>It is also useful to think about the statistical methods you will use
to analyse the data. If you’re planning to bring a data analyst or
bioinformatician onboard for data analysis, you should include him or
her in the experimental design stage.</p></section><section id="aio-Terminology"><p>Content from <a href="Terminology.html">Terminology</a></p>
<hr>
<p>Last updated on 2025-08-12 |

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/episodes/Terminology.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 65 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<p>Before progressing, it may be useful to define some terms which are
commonly used in RNA-seq.</p>
<table class="table">
<colgroup>
<col width="33%">
<col width="66%">
</colgroup>
<thead><tr class="header">
<th>Term</th>
<th>Definition</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><strong>Variability:</strong></td>
<td>A measure of how much the data is spread around. Variance is
mathematically defined as the average of the squared difference between
observations and the expected value. Simply put, a larger variance means
it is harder to identify differentially expressed genes.</td>
</tr>
<tr class="even">
<td><strong>Feature:</strong></td>
<td>A defined genomic region. Usually a gene in RNA-seq, but can also
refer to any region such as an exon or an isoform. In RNA-seq, an
estimate of abundance is obtained for each feature.</td>
</tr>
<tr class="odd">
<td><strong>Biological replicates:</strong></td>
<td>Samples that have been obtained from biologically separate samples.
This can mean different individual organisms (e.g. tissue samples from
different mice), different samplings of the same tumour, or different
population of cells grown separately from each other but originating
from the same cell-line. For example, the samples obtained from three
different knock-out mice could be considered biological replicates in a
knock-out versus wild-type experiment. A biological replicate combines
both technical and biological variability as it is also an independent
case of all the technical steps.</td>
</tr>
<tr class="even">
<td><strong>Technical replicates:</strong></td>
<td>Samples in which the starting biological sample is the same, but the
replicates are processed separately. For example, if a biological sample
is divided and two different library preps are processed and sequenced,
those two samples would be considered technical replicates.</td>
</tr>
<tr class="odd">
<td><strong>Covariate:</strong></td>
<td>The term ‘covariate’ is often used interchangeably with ‘factor’ or
‘variable’ in RNA-seq. The term refers to a property of the sample which
may have some influence on gene expression and should be represented in
the RNA-seq model. Covariates in RNA-seq are often categorical
(e.g. treatment condition, sex, batch), but continuous factors are also
possible (e.g. time points, age). A linear model will contain terms to
represent the relationships between covariates and each sample. Each
possible value a factor can take is called a level (e.g. ‘male’ and
‘female’ are two levels in the factor ‘sex’). Factors can either be
directly of interest to the experiment (e.g. treatment condition) or not
of interest (also known as nuisance variables) (e.g. sex, batch). The
purpose of covariates is to explain the variance seen in samples.</td>
</tr>
<tr class="even">
<td><strong>Confounding variable:</strong></td>
<td>A confounding variable is a nuisance variable that is associated
with the factor of interest. Possible confounding factors should be
controlled for so they don’t interfere with analysis. For example, if
all knock-out mice samples were harvested in the morning and all
wild-type mice samples were harvested in the afternoon, the time of
sample collection would be a confounding factor as the effects from
sample collection time and from the knock-out cannot be separated.</td>
</tr>
<tr class="odd">
<td><strong>Statistical power:</strong></td>
<td>The ability to identify differentially expressed genes when there
really is a difference. This is partly dependent on variance and
therefore is affected by the number of replicates available and
sequencing depth.</td>
</tr>
</tbody>
</table></section><section id="aio-The importance of replicates to estimate variance"><p>Content from <a href="The%20importance%20of%20replicates%20to%20estimate%20variance.html">The importance of replicates to estimate variance</a></p>
<hr>
<p>Last updated on 2025-08-12 |

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/episodes/The%20importance%20of%20replicates%20to%20estimate%20variance.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 65 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<p>When performing a differential gene expression analysis, we look at
the expression values of each gene and try to determine if the
expression is significantly different in the different conditions
(e.g. knock-out and wild-type). The ability to distinguish whether a
gene is differentially expressed is partly determined by the estimates
of variability obtained by using multiple observations in each
condition.</p>
<p>Variability is present in two forms: technical variability and
biological variability.</p>
<p>Combined biological and technical variability is measured using
biological replicates. Biological variability is the main source of
variability and is due to natural variation in the population and within
cells. This includes different individuals having different levels of a
particular gene and the stochastic nature of expression levels in
different cells.</p>
<p>Technical variability is measured using technical replicates.
Technical variability is often very small compared to biological
variability. Usually the question is whether an observed difference is
greater than the total variability (i.e. significant). As combined
variability is measured by biological replicates technical replicates
are only important if you need to know the degree of biological
variability or technical variability. An example of wanting technical
variability would be method development. The main source of technical
variation comes from RNA processing and from library prep. Variability
from sequencing in different flow cells or different lanes is usually
minimal. Generally, creating technical replicates from multiple library
preps is unnecessary for RNA-seq experiments.</p>
<p>The amount of variance between your biological replicates will affect
the outcome of your analysis. Ideally, you aim to have minimal
variability between samples so you only measure the effect of the
condition of interest. Too much variability between samples can drown
out the signal of truly differentially expressed genes. Controlling for
possible confounding factors between conditions is also important to
prevent falsely attributing differential expression to the condition of
interest.</p>
<p>Strategies to minimise variation between samples and to control
confounding variables include:</p>
<ul>
<li>choosing organisms from the same litter,</li>
<li>choosing organisms of the same sex if possible,</li>
<li>using a constant sample collection time,</li>
<li>having the same laboratory technician perform each library
prep,</li>
<li>randomising samples to prevent a confounding batch effect if all
samples can’t be processed at one time.</li>
</ul>
<p>If variation between samples can not be removed it should be balanced
between conditions of interest as much as possible, and carefully
recorded to allow its effect to be measured and potentially removed
during analysis.</p></section><section id="aio-How many replicates and how many reads do I need"><p>Content from <a href="How%20many%20replicates%20and%20how%20many%20reads%20do%20I%20need.html">How many replicates and how many reads do I need?</a></p>
<hr>
<p>Last updated on 2025-08-12 |

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/episodes/How%20many%20replicates%20and%20how%20many%20reads%20do%20I%20need.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 65 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<p>Two very common question asked are:</p>
<ul>
<li>how many biological replicates do I need, and</li>
<li>what sequencing depth is needed for each sample</li>
</ul>
<p>in order to have enough statistical power for my RNA-seq
experiment?</p>
<p>These questions cannot be precisely answered without a pilot study. A
small amount of data (minimum of two biological replicates for each
condition with at least 10M reads) can estimate the amount of biological
variation, which determines how many biological replicates are required.
Performing a pilot study is highly recommended to estimate statistical
power and identify possible problems before investing more time and
money into the project.</p>
<p><a href="http://euler.bc.edu/marthlab/scotty/scotty.php" class="external-link">Scotty</a>
is a web-based tool that uses data generated from a pilot study to
optimize a design for statistical power. With a limited budget, one must
balance sequence coverage and number of biological replicates. Scotty
also has a cost estimate feature which returns the most powerful design
within budget constraints.</p>
<p>As a general rule, the number of biological replicates should never
be below 3. For a basic RNA-seq differential expression experiment, 10M
to 20M reads per sample is usually enough. If similar data exists it can
be helpful to check the read counts for key genes of interest to
estimate the required depth.</p>
<p>Biological variability is usually the largest effect limiting the
power of RNA-seq analysis. The most improvement in an experiment will
usually be achieved by increasing the biological replication to improve
estimation of the biological variation.</p>
<p>It is often possible to design experiments where the analysis is done
incrementally such that a pilot study is added to with an additional
block of samples or a pool of libraries is sequenced to additional
depth. In these cases care must be taken to balance the design in a
manner that each stage is a valid experiment in its own right. This can
allow a focused question to be answered in the first stage, with an
ability to either address issues or progress to a second stage with
additional questions.</p></section><section id="aio-Sequencing options to consider"><p>Content from <a href="Sequencing%20options%20to%20consider.html">Sequencing options to consider</a></p>
<hr>
<p>Last updated on 2025-08-12 |

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/episodes/Sequencing%20options%20to%20consider.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 65 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<p><strong>How much total RNA is needed:</strong> Many sequencing
centres such as AGRF recommend at least 250ng of total RNA for RNA
sequencing. It is possible to go as low as 100ng of total RNA, but
results are not guaranteed. The quality of RNA is also important when
making libraries. A RNA Integrity Number (RIN) is a number from 1 (poor)
to 10 (good) and can indicate how much degradation there is in the
sample. A poor score can lead to over representation at the 3’ end of
the transcript and low yield. Samples with low RIN scores (below 8) are
not recommended for sequencing. Care should also be taken to ensure RIN
is consistent between conditions to avoid confounding this technical
effect with the biological question.</p>
<p><strong>Choosing an enrichment method:</strong> Ribosomal RNA makes
up &gt;95% of total cellular RNA, so a preparation for RNA-seq must
either enrich for mRNA using poly-A enrichment, or deplete rRNA. Poly-A
enrichment is recommended for most standard RNA-seq experiments, but
will not provide information about microRNAs and other non-coding RNA
species. In general, ribo-depleted RNA-seq data will contain more noise,
however, the protocol is recommended if you have poor or variable
quality of RNA as the 3’ bias of poly-A enrichment will be more
pronounced with increased RNA degradation. The amount of RNA needed for
each method differs. For Poly-A enrichment a minimum of 100ng is needed
while for ribo-depletion, a minimum of 200ng is recommended.</p>
<p><strong>Choosing read type:</strong> For basic differential
expression analysis RNA-seq experiments, single-end sequencing is
recommended to obtain gene transcript counts. In more advanced
experiments, paired-ends are useful for determining transcript structure
and discovering splice variants.</p>
<p><strong>Choosing strandedness:</strong> With a non-directional
(unstranded) protocol, there is no way to identify whether a read
originated from the coding strand or its reverse complement.
Non-directional protocols allow mapping of a read to a genomic location,
but not the direction in which the RNA was transcribed. They are
therefore used to count transcripts for known genes, and are recommended
for basic RNA-seq experiments. Directional protocols (stranded) preserve
strand information and are useful for novel transcript discovery.</p>
<p><strong>Multiplexing:</strong> Multiplexing is an approach to
sequence multiple samples in the same sequencing lane. By sequencing all
samples in the same lane, multiplexing can also minimise bias from lane
effects.</p>
<p><strong>Spike-in controls:</strong> RNA-seq spike-in controls are a
set of synthetic RNAs of known concentration which act as negative or
positive controls. These controls have been used for normalisation and
quality control, but recent work has shown that the amount of technical
variability in their use dramatically reduces their utility.</p></section><section id="aio-Summary"><p>Content from <a href="Summary.html">Summary</a></p>
<hr>
<p>Last updated on 2025-08-12 |

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/episodes/Summary.Rmd" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 65 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<ul>
<li><p>A good experimental design is vital for a successful experiment.
If you’re planning to work with a data analyst or bioinformatician,
include them in the design stage.</p></li>
<li><p>Aim to minimise variability by identifying possible sources of
variance in your samples.</p></li>
<li><p>Biological replicates are important. The number of biological
replicates you should have should never be below 3. Technical replicates
are often unnecessary.</p></li>
<li><p>Pilot studies are highly recommended for identifying how many
replicates and how many reads you should have for enough statistical
power in your experiment.</p></li>
<li><p>For basic RNA-seq experiments, poly-A enriched, single-ended,
unstranded sequencing at depths of 10M to 20M is probably what you
want.</p></li>
</ul></section>
</div>
    </main>
</div>
<!-- END  : inst/pkgdown/templates/content-extra.html -->

      </div>
<!--/div.row-->
      		<footer class="row footer mx-md-3"><hr>
<div class="col-md-6">
        <p>This lesson is subject to the <a href="CODE_OF_CONDUCT.html">Code of Conduct</a></p>
        <p>

        <a href="https://github.com/melbournebioinformatics/workbench-template-md/edit/main/README.md" class="external-link">Edit on GitHub</a>

	
        | <a href="https://github.com/melbournebioinformatics/workbench-template-md/blob/main/CONTRIBUTING.md" class="external-link">Contributing</a>
        | <a href="https://github.com/melbournebioinformatics/workbench-template-md/" class="external-link">Source</a></p>
				<p><a href="https://github.com/melbournebioinformatics/workbench-template-md/blob/main/CITATION.cff" class="external-link">Cite</a> | <a href="mailto:fixme@fixme.org">Contact</a> | <a href="https://carpentries.org/about/" class="external-link">About</a></p>
			</div>
			<div class="col-md-6">

        <p>Materials licensed under <a href="LICENSE.html">CC-BY 4.0</a> by the authors</p>

        <p>Template licensed under <a href="https://creativecommons.org/licenses/by-sa/4.0/" class="external-link">CC-BY 4.0</a> by <a href="https://carpentries.org/" class="external-link">The Carpentries</a></p>
        <p>Built with <a href="https://github.com/carpentries/sandpaper/tree/0.17.1" class="external-link">sandpaper (0.17.1)</a>, <a href="https://github.com/carpentries/pegboard/tree/0.7.9" class="external-link">pegboard (0.7.9)</a>, and <a href="https://github.com/melbournebioinformatics/uom-varnish/tree/main" class="external-link">varnish (1.0.7.9000)</a></p>
			</div>
		</footer>
</div> <!-- / div.container -->
	<div id="to-top">
		<a href="#top">
      <i class="search-icon" data-feather="arrow-up" role="img" aria-label="Back To Top"></i><br><!-- <span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top --><span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top
		</a>
	</div>
  <script type="application/ld+json">
    {
  "@context": "https://schema.org",
  "@type": "LearningResource",
  "@id": "melbournebioinformatics.github.io/uom-varnish/instructor/aio.html",
  "inLanguage": "en",
  "dct:conformsTo": "https://bioschemas.org/profiles/LearningResource/1.0-RELEASE",
  "description": "A Carpentries Lesson teaching foundational data and coding skills to researchers worldwide",
  "keywords": "software, data, lesson",
  "name": "All in One View",
  "creativeWorkStatus": "active",
  "url": "melbournebioinformatics.github.io/uom-varnish/instructor/aio.html",
  "identifier": "melbournebioinformatics.github.io/uom-varnish/instructor/aio.html",
  "dateCreated": "2025-07-26",
  "dateModified": "2025-08-12",
  "datePublished": "2025-08-12"
}

  </script><script>
		feather.replace();
	</script>
</body>
</html><!-- END:   inst/pkgdown/templates/layout.html-->

